docs/topics/spiders.rst

```diff
@@ -169,6 +169,14 @@ scrapy.Spider
                    # each of them, with another callback
                    pass
 
+       .. note::
+           .. versionchanged:: 2.x
+
+           By default the requests produced by :meth:`start_requests` can be
+           processed much later than expected. For :meth:`start_requests`
+           declared as an ``async def`` function a better behavior is used. See
+           :ref:`async-start_requests-queue` for details.
```

Comment:
I'm not sure "default" is a right word here. For `def start_requests` behavior is this, for `async def start_requests` behavior is that.

I guess here we're suggesting to use `def start_requests` becuase `async def start_requests` could break some third-party middlewares, is this true? Should we explain it here? Otherwise it looks a bit strange - we tell that the example is not written in the best way, but we're not explaining why.

Reply:
I also wonder how to word it to make sure `versionchanged` above makes sense. start_requests behavior is not getting worse in 2.4, it is the opposite - there is now a way around it :)

Reply:
This note is not for the example but for the `.. method:: start_requests()` block as this is the only place where it can be written except the coroutines package that it links to. I didn't intend to suggest anything, just to write here that there are two possible behaviors.